sled-agent/config-reconciler/test-data/expectorate/merged-sled-config.json

```diff
@@ -809,5 +809,6 @@
         "type": "install_dataset"
       }
     }
-  }
+  },
+  "remove_mupdate_override": null
```

Comment:
Are there backcompat considerations here?

Reply:
Definitely not yet - nothing has landed or shipped that would ledger these merged configs on deployed systems. (Even if we had, I think the answer would still be no - this is confirming that we can convert the legacy configs to the new config, and getting `null` here is exactly what we want.)